pkg/eventmonitor/config/config.go

```diff
@@ -0,0 +1,282 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	"fmt"
+
+	"os"
+
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	coreconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/ebpf"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	rsNS = "runtime_security_config"
+	evNS = "event_monitoring_config"
+)
+
+type Config struct {
+	ebpf.Config
+
+	// EnableKernelFilters defines if in-kernel filtering should be activated or not
+	EnableKernelFilters bool
+
+	// EnableApprovers defines if in-kernel approvers should be activated or not
+	EnableApprovers bool
+
+	// EnableDiscarders defines if in-kernel discarders should be activated or not
+	EnableDiscarders bool
+
+	// FlushDiscarderWindow defines the maximum time window for discarders removal.
+	// This is used during reload to avoid removing all the discarders at the same time.
+	FlushDiscarderWindow int
+
+	// SocketPath is the path to the socket that is used to communicate with the security agent and process agent
+	SocketPath string
```

Comment:
Unless I missed something, this one is only used by the `process` consumer no ? If so, should we move this parameter away from this package (as it is consumer specific), and into a separate config for the process consumer ?

Reply:
Had a chat with @safchain about this one, it might be OK for now, we'll revisit in the future if need be.